### PR TITLE
Feature/create places collection

### PR DESCRIPTION
**Pull request by dav3r**

### CODE REVIEW

**Review by felddy — APPROVED** [review]
@dav3r what is the state of the scripts you are importing from the private repo?  Is there any special sauce there worth protecting?  I see a lot of hoops that you're jumping through to get those scripts here.  I wonder if they can't just live in the "files" of the cyhy_commander ansible role and be copied over.  Eventually we'd want this so other people can use it.  Maybe now is the time to move them and simplify this?

File: ansible/roles/cyhy_commander/tasks/main.yml [hunk 1]
```diff
@@ -98,3 +98,91 @@
...
+    path: /tmp/cyhy-places/scripts
+    state: directory
+
+- name: Create the /tmp/cyhy-places/extras directory
+  file:
+    path: /tmp/cyhy-places/extras
+    state: directory
+
+- name: Download load_places.sh from GitHub
+  shell: "curl -o /tmp/cyhy-places/scripts/load_places.sh https://{{ github_oauth_token }}@raw.githubusercontent.com/jsf9k/cyhy-core/develop/var/load_places.sh"
```
Comment (jsf9k) [inline ansible/roles/cyhy_commander/tasks/main.yml]: You could use the Ansible module [`get_url`](https://docs.ansible.com/ansible/latest/modules/get_url_module.html) here and for the two similar lines below.  I think that's a little cleaner than `shell`.

File: ansible/roles/cyhy_commander/tasks/main.yml [hunk 2]
```diff
@@ -98,3 +98,91 @@
...
+
+- name: Download GNIS_data_import.py from GitHub
+  shell: "curl -o /tmp/cyhy-places/scripts/GNIS_data_import.py https://{{ github_oauth_token }}@raw.githubusercontent.com/jsf9k/cyhy-core/develop/var/GNIS_data_import.py"
```
Comment (jsf9k) [inline ansible/roles/cyhy_commander/tasks/main.yml]: See comment above.

File: ansible/roles/cyhy_commander/tasks/main.yml [hunk 3]
```diff
@@ -98,3 +98,91 @@
...
+    path: /tmp/cyhy-places/scripts/load_places.sh
+    mode: 0755
+
+- name: Make GNIS_data_import.py script executable
+  file:
+    path: /tmp/cyhy-places/scripts/GNIS_data_import.py
+    mode: 0755
+
+- name: Download ADDL_CYHY_PLACES.txt from GitHub
+  shell: "curl -o /tmp/cyhy-places/extras/ADDL_CYHY_PLACES.txt https://{{ github_oauth_token }}@raw.githubusercontent.com/jsf9k/cyhy-core/develop/extras/ADDL_CYHY_PLACES.txt"
```
Comment (jsf9k) [inline ansible/roles/cyhy_commander/tasks/main.yml]: See comment above.

**Review by jsf9k — APPROVED** [review]
This looks good to me.

**Comment by jsf9k** [comment]
I'll go ahead and change those `shell` commands to use `get_url` instead.  If I don't get it working by the time I stop for the night I'll go ahead and merge this.

**Comment by dav3r** [comment]
@felddy I agree that these files will be useful to others, but I think we should hold off on that until we split this repo into two (creating the AMIs vs. everything that happens after that).

@jsf9k - nice work; I see you didn't completely ignore the GitHub docs like I did. 👍

**Comment by jsf9k** [comment]
I think this is in very good shape now.  If others agree with the changes I made, please go ahead and merge.

**Comment by jsf9k** [comment]
@dav3r 

"If I have seen further it is by standing on the shoulders of giants."  --Sir Issac Newton

**Pull request merged**